src/style.css

```diff
@@ -0,0 +1,21 @@
+.mg-t {
+  margin-top: 1rem;
```

Comment:
IMO, using `rem` for margin is not good practice

Reply:
What should unit we use?

Reply:
you can use `px`.